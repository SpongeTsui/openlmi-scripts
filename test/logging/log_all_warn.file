WARNING: Command "lmi.scripts.cmdtest.Show" is missing usage string. It will be inherited from parent command.
WARNING: One Ring to bring them all
ERROR: and in the darkness bind them.
CRITICAL: In the land of Mordor where the Shadows lie.
