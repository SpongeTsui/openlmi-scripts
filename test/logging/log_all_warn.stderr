warning : Command "lmi.scripts.cmdtest.Show" is missing usage string. It will be inherited from parent command.
warning : One Ring to bring them all
error   : and in the darkness bind them.
critical: In the land of Mordor where the Shadows lie.
